src/Umbraco.Web.UI.Client/src/packages/core/entity-action/common/delete/types.ts

```diff
@@ -8,6 +8,10 @@ export interface ManifestEntityActionDeleteKind extends ManifestEntityAction<Met
 export interface MetaEntityActionDeleteKind extends MetaEntityActionDefaultKind {
 	detailRepositoryAlias: string;
 	itemRepositoryAlias: string;
+	confirm?: {
+		headline?: string;
+		content?: string;
```

Comment:
I think we have other places where the wording is `message` or `description`. Content feels a bit too wide :-)